tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -90,7 +90,8 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 			scParameters[scParamStoragePolicyID] = profileID
```

Comment:
move this block together with SC creation

Reply:
Updated the code for all three flavours .